requirements/app.in

```diff
@@ -53,8 +53,8 @@ marshmallow-sqlalchemy>=0.23.1
 webargs
 # Minimum version that correctly aligns time series that include NaN values
 uniplot>=0.7.0
+# Maximum constraints here due to Flask-Classful not supporting Werkzeug 2.2.0 yet, see https://github.com/teracyhq/flask-classful/pull/145
+Flask-SQLAlchemy>=2.4.3,<3
 # flask should be after all the flask plugins, because setup might find they ARE flask
-# Minimum here due to Flask-Classful not supporting Werkzeug 2.2.0 yet, see https://github.com/teracyhq/flask-classful/pull/145
-flask>=1.0, <=2.1.2
-# remove if flask max constraint (see above) is removed 
```

Comment:
Why remove the comment but not the pin?

Reply:
This only needed one comment (to talk about multiple contraints), and the correct word is "Maximum".